utils/constants.py

```diff
@@ -34,6 +34,7 @@
     'bugzilla-qe-verify',
     'confluence-updates',
     'github-issue-regression',
+    'sentry-top-issues'
```

Comment:
not sure about this name... what we are trying to get are the new crashes not tracked yet by devs so that they are aware of them and could take them to fix them... 
For me, top issues would be those crashes with more frequency. That could be something to track also in the future